src/audio3d/Audio3DConverts.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __PIROGRONIAN__AUDIO_3D__CONVERTS__H__
+#define __PIROGRONIAN__AUDIO_3D__CONVERTS__H__
+
+#include <Eigen/Core>
+#include <AL/al.h>
+
+namespace Audio3D 
+{
+
+    Eigen::Vector3d inline convert(const ALfloat *v)
+    {
+        return Eigen::Vector3d(v[0], v[1], v[2]);
```

Comment:
no bounds checking?

Reply:
and `Eigen::Map<Eigen::Vecto3f>(v)` will be better

Reply:
I want keep it as simple (and fast) as possible. It isn't intended to be used outside of audio module.